packages/web-components/src/components/card/card.ts

```diff
@@ -112,52 +128,52 @@ class DDSCard extends StableSelectorMixin(BXLink) {
    * @returns The inner content.
    */
   protected _renderInner() {
-    const {
-      _handleSlotChange: handleSlotChange,
-      _hasPictogram: hasPictogram,
-      _hasCopy: hasCopy,
-    } = this;
-    return html`
-      ${this._renderImage()}
-      <div
-        class="${prefix}--card__wrapper ${hasPictogram
-          ? `${prefix}--card__pictogram`
-          : ''} ${hasPictogram && hasCopy ? `${prefix}--card__motion` : ''}">
-        <div class="${prefix}--card__content">
-          ${hasPictogram ? '' : html` <slot name="eyebrow"></slot> `}
-          ${this.pictogramPlacement === PICTOGRAM_PLACEMENT.TOP
-            ? html`
-                <slot
-                  name="pictogram"
-                  data-pictogram-placement="${PICTOGRAM_PLACEMENT.TOP}"
-                  @slotchange="${handleSlotChange}"></slot>
-              `
-            : ''}
-          ${this.pictogramPlacement !== PICTOGRAM_PLACEMENT.TOP || !hasPictogram
-            ? this._renderHeading()
-            : null}
-          ${this.pictogramPlacement === PICTOGRAM_PLACEMENT.BOTTOM ||
-          !hasPictogram
-            ? this._renderCopy()
-            : ''}
-          ${this.pictogramPlacement === PICTOGRAM_PLACEMENT.BOTTOM
-            ? html`
-                <slot
-                  name="pictogram"
-                  data-pictogram-placement="${PICTOGRAM_PLACEMENT.BOTTOM}"
-                  @slotchange="${handleSlotChange}"></slot>
-              `
-            : ''}
-          ${hasPictogram && this.pictogramPlacement === PICTOGRAM_PLACEMENT.TOP
-            ? this._renderHeading()
-            : null}
-          ${hasPictogram && this.pictogramPlacement === PICTOGRAM_PLACEMENT.TOP
-            ? this._renderCopy()
-            : ''}
-          <slot name="footer"></slot>
+    const { _hasPictogram: hasPictogram, _hasCopy: hasCopy } = this;
+    if (hasPictogram) {
+      return html`
+        ${this._renderImage()}
+        <a
+          tabindex="0"
+          href="${this.href}"
+          target="_self"
+          class="${prefix}--card__wrapper ${prefix}--card__pictogram
+           ${hasCopy ? `${prefix}--card__motion` : ''}">
+          <div class="${prefix}--card__content">
+            ${this.pictogramPlacement === PICTOGRAM_PLACEMENT.TOP
+              ? this._renderPictogram('top')
+              : ''}
+            ${this.pictogramPlacement !== PICTOGRAM_PLACEMENT.TOP ||
+            !hasPictogram
+              ? this._renderHeading()
+              : null}
+            ${this.pictogramPlacement === PICTOGRAM_PLACEMENT.BOTTOM ||
+            !hasPictogram
+              ? this._renderCopy()
+              : ''}
+            ${this.pictogramPlacement === PICTOGRAM_PLACEMENT.BOTTOM
+              ? this._renderPictogram('bottom')
```

Comment:
Same thing here about using the enum.
```suggestion
              ? this._renderPictogram(PICTOGRAM_PLACEMENT.BOTTOM)
```